### PR TITLE
sandbox form prototype

### PR DESCRIPTION
**Pull request by dgolovin**
- POC to provision sandbox
- Prototype of Sandbox integration in Add Cluster editor

### CODE REVIEW

File: src/webview/cluster/app/cluster.tsx [hunk 1]
```diff
@@ -60,15 +61,16 @@ const vscodeApi = window.vscodeApi;
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 export default function Header() {
   const classes = useStyles();
-  const [showWizard, setShowWizard] = React.useState(false);
+  const [showWizard, setShowWizard] = React.useState('');
 
   const handleView = (index) => {
     if (index === 0) {
-      setShowWizard(!showWizard);
```
Comment (gorkem) [inline src/webview/cluster/app/cluster.tsx:67]: Better to make this `if` a `switch`.

**Comment by mohitsuman** [comment]
@dgolovin I have fixed the UI for the workflow in the latest 2 commits. The elements to select phone number is through a drop-down now.

**Comment by mohitsuman** [comment]
We might need to test the phone verification properly again after the changes. And there is one issue in the workflow, if the user during `Login` cancels the VSCode pop up (coming from vscode-redhat-authentication) then the spinner is stuck throughout. We need to close the editor and start again.

**Comment by mohitsuman** [comment]
This is how the first page of the workflow will look. All are working links.

Screenshot 2022-01-24 at 12 30 26 AM

**Comment by mohitsuman** [comment]
For the user profile, the information will refer to the currently logged-in user session.

Screenshot 2022-01-24 at 1 53 50 AM

**Pull request merged**